lingua_franca/lang/parse_nl.py

```diff
@@ -560,9 +561,10 @@ def date_found():
                 minAbs or secOffset != 0
             )
 
-    if text == "" or not anchorDate:
+    if text == "":
         return None
 
+    anchorDate = anchorDate or now_local()
     found = False
     daySpecified = False
     dayOffset = False
```

Comment:
because thats how it was in mycroft-core, #97 did a major cleanup of this, but theres still some places to go. essentially the original english parser used CamelCase and the rest just copy pasted it

Reply:
Ok, guess we can table this for another time.